test/k8s/ci_assets/nuctl.sh

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env bash
+
+set -e
```

Comment:
```suggestion
# exit on failure
set -o errexit

# show command before execute
set -o xtrace
```

Reply:
I added xtrace so ci will show the executed commands + flow

It might add some noise but will surely help debug the executed commands

Reply:
`set -e` is the same as `set -o errexit` and very prevalent... but sure.
About the xtrace - I'm afraid that messes sup the output too much (prints the `echo`s)
since the commands are in plane bash, and not some dynamically generated cryptic commands and flags - I believe this is not necessary here.